ladybug_comfort/adaptive.py

```diff
@@ -80,53 +60,108 @@ def adaptive_comfort_en15251(t_prevail, ta, tr, vel, conditioning=0):
             comfort model, this is typically the exponentially weighted running mean
             of the outdoor temperature over the past week.  Use the weighted_running_mean
             functions to compute this value.
-        ta: Air temperature [C]
-        tr: Mean radiant temperature [C]
-        vel: Relative air velocity [m/s]
+        to: Operative temperature [C]
+
+    Returns:
+        result: A dictionary containing results with the following keys:
+            to : Operative Temperature [C].
+            t_comf : Adaptive comfort neutral temperature (desired by occupants) [C].
+            deg_comf: The difference between the operative temperature (to)
+                and the adaptive comfort neutral temperature (t_comf) [C].
+                Negative values inidcate cool conditions and positive values
+                indicate varm conditions.
+    """
+    # fix upper and lower outdoor temperatures if outside the range of the model
+    if t_prevail < 10.0:
```

Comment:
I understand that giving user feedback is a good idea but I do not think that this is the right place for a warning.  This function will be run hundreds of thousands of times when building the comfort maps and hundreds of thousands of print messages saying the same thing isn't useful to anyone.  It will only stop people from seeing the real warnings and errors that they actually need to worry about.

Also, if it is not clear, this is how adaptive comfort methods are supposed to work.  Once the temperature is beyond certain limits, the comfort range becomes fixed.  This isn't to say that there is still not merit in letting people know when they have a data set that goes beyond these limits.  It is just that a utility function is not the place to do it.

Reply:
I see your point. Can we then add a method that just runs the check once and gives a report on the input data before calling this function.

Reply:
A method like that sounds very useful.  It will let us be flexible about how we give the user feedback and we can give them a message like `x prevailing outdoor values do not fall within the adaptive comfort range` instead of just giving them many print messages.  I will implement this now.

Reply:
Done!  I am adding tests for the new method now.